experimental/addons/crane/metadata.yaml

```diff
@@ -1,18 +1,23 @@
 name: crane
-version: v0.5.0
-description: crane
-icon: https://docs.gocrane.io/dev/images/crane.svg
-url:
 
-tags:
-  - crane
+version: 0.5.0
+
+description: Crane (FinOps Crane) is an open-source project which manages cloud resource on Kubernetes stack, it is inspired by FinOps concepts.
+
+icon: https://raw.githubusercontent.com/gocrane/crane/main/docs/images/crane.png
 
-deployTo:
-  control_plane: true
-  runtime_cluster: false
+url: https://docs.gocrane.io/
+
+invisible: false
 
 dependencies:
-  - name: velaux
-  - name: fluxcd
+- name: fluxcd
 
```

Comment:
Since this an in-cluster controller, do we need install this in runtime controller.